packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -93,9 +93,32 @@ ZIP_PREFIX=instana-nodejs-layer
 ZIP_NAME=$ZIP_PREFIX.zip
 TMP_ZIP_DIR=tmp
 
-echo Will build Lambda layer with name \"$LAYER_NAME\".
+if [[ -z $REGION ]]; then
+  REGIONS=$'ap-northeast-1\nap-northeast-2\nap-south-1\nap-southeast-1\nap-southeast-2\nca-central-1\neu-central-1\neu-north-1\neu-west-1\neu-west-2\neu-west-3\nsa-east-1\nus-east-1\nus-east-2\nus-west-1\nus-west-2'
+else
+  REGIONS=$REGION
+fi
 
-REGIONS=$'ap-northeast-1\nap-northeast-2\nap-south-1\nap-southeast-1\nap-southeast-2\nca-central-1\neu-central-1\neu-north-1\neu-west-1\neu-west-2\neu-west-3\nsa-east-1\nus-east-1\nus-east-2\nus-west-1\nus-west-2'
+while true; do
+    echo "####"
+    echo "LAYER_NAME: $LAYER_NAME"
+    echo "SKIP_DOCKER_IMAGE: $SKIP_DOCKER_IMAGE"
+    echo "DOCKER_IMAGE_NAME: $DOCKER_IMAGE_NAME"
+    echo "REGIONS: $REGIONS"
+    echo "PACKAGE_VERSION: $PACKAGE_VERSION"
+    echo "BUILD_LAYER_WITH: $BUILD_LAYER_WITH"
+    echo "SKIP_AWS_PUBLISH_LAYER: $SKIP_AWS_PUBLISH_LAYER"
+    echo "####"
+
+    read -p "Do you wish to continue (yes or no)? " yn
+    case $yn in
+        [Yy]* ) echo "Let's go!"; break;;
+        [Nn]* ) exit 1;;
+        * ) echo "Please answer yes or no.";;
+    esac
+done
```

Comment:
_issue:_ That's pretty nice for local use 👍    One caveat: On [Concourse](https://github.com/instana/nodejs/blob/main/packages/serverless/ci/pipeline.yaml#L310-L315), nobody will press `y`, so I think we need a parameter or env var to skip the question for usage on CI.

Reply:
Yeah good catch! I'll skip